examples/federation/README.adoc

```diff
@@ -0,0 +1,222 @@
+== Coherence Operator Federation Example
+
+This simple example demostrates the Coherence federation feature.  It shows how to deploy two Coherence clusters that federating data between them using the Coherence Operator. The Coherence federation feature requires Coherence Grid Edition.
+
+You can find the source code in the https://github.com/oracle/coherence-operator/tree/master/examples/federation[Operator GitHub Repo].
+
+* <<pre,Prerequisites>>
+* <<build, Build the Example>>
+* <<example, Run the Example>>
+* <<cleanup, Cleaning Up>>
+
+[#pre]
+== Prerequisites
+
+Following the instructions in https://github.com/oracle/coherence-operator/tree/master/examples/deployment[Coherence Operator Deployment Example] for all the prerequisites and installing the Coherence Operator. 
+
+Confirm the operator is running, for example:
+[source,bash]
+----
+kubectl get pods -n coherence-example
+
+NAME                                                     READY   STATUS    RESTARTS   AGE
+coherence-operator-controller-manager-74d49cd9f9-sgzjr   1/1     Running   1          27s
+----
+
+[#build]
+== Build the Example
```

Comment:
The whole build section can be removed. We do not need to build anything as we can just use the commercial Coherence image from `container-registry.oracle.com/middleware/coherence:14.1.1.0.0` and you then provide the configuration using the config secret.

You probably need to add instructions on how to get this image (see https://oracle.github.io/coherence-operator/docs/latest/#/installation/04_obtain_coherence_images)

Then they need to:

- Create the `coherence-example` namespace
- Create an image pull secret named `ocr-pull-secret` containing their OCR credentials (this is then used in the Coherence yaml files).